common/src/main/java/com/mx/common/exception/README.md

```diff
@@ -0,0 +1,49 @@
+# Path Exceptions
+
+Throwing the correct exception is vital for the Path system to behave in an expected way. Use of more specific exceptions that is encouraged. 
```

Comment:
Grammar. `Use of more specific exceptions is encouraged.`

Reply:
Fixed.